README.md

```diff
@@ -16,29 +19,42 @@ An elemental block to enable more control over the responsiveness of tables.
 
 ## Usage
 
-### Content editable fields:
+### Content editable fields
+
 * Title
 * Description field
 * Caption field
 
-### Settings:
+### Settings
+
 * Set the Number of Columns
 * Set the Number of Rows
 * Provide the CSS classes for alignments and styles, e.g. "Zebra rows"
 * Set the width for each column
 
-### Enable RichLinks for CWP site:
+### Enable RichLinks for CWP site
+
 In your ```_config.yml``` file
-```
+
+```yml
 Signify\Factory\Models\TableBlock:
   enable_rich_links: true
 ```
 
-### Styles:
+### Styles
+
 You need to add CSS styles on your site to theme the table block.
 
 ### Screenshots
+
+Main content
 ![content](docs/en/img/content.png)
+
+Edit cells
 ![content](docs/en/img/table-items.png)
+
+Settings
```

Comment:
The two "settings" images should be merged into one, and whitespace on the right reduced, to make the text bigger and easier to read.

That doesn't have to be done this in this pull request, it's mainly a note for when you're producing documentation in future.